cluster/examples/kubernetes/ceph/upgrade-from-v1.1-crds.yaml

```diff
@@ -191,27 +188,6 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
-metadata:
```

Comment:
Why are you removing the net-new `CephClient` CRD?

Reply:
This moved to the `upgrade-from-v1.1-apply.yaml`, which will now be applied in step 1 before the upgrade. New CRDs have to be created before the upgrade.